gvr-widgetviewer/app/build.gradle

```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
+

```

Comment:
jsut be consistent with the module name, would it be better?
gvr-widgetviewer/app/build.gradle => gvr-widgetviewer/gvrwidgetviewer/build.gradle 

The same with settings.gradle